Dockerfile

```diff
@@ -550,12 +596,12 @@ COPY --chown=0:0 --from=sdk-rust \
   /usr/share/licenses/rust/
 
 # "sdk-go" has the Go toolchain and standard library builds.
-COPY --chown=0:0 --from=sdk-go /home/builder/go/bin /usr/libexec/go/bin/
-COPY --chown=0:0 --from=sdk-go /home/builder/go/lib /usr/libexec/go/lib/
-COPY --chown=0:0 --from=sdk-go /home/builder/go/pkg /usr/libexec/go/pkg/
-COPY --chown=0:0 --from=sdk-go /home/builder/go/src /usr/libexec/go/src/
+COPY --chown=0:0 --from=sdk-go /home/builder/sdk-go/bin /usr/libexec/go/bin/
```

Comment:
I'm okay with this change, but I'm less certain this should accompany the `govc` addition. 

If no one else has objections, then it can come in this way, but I'd prefer to have seen this in its own PR (even if we'd have to review & merge that before `govc`).

Reply:
#55 has the sdk-go changes separately now - will move discussion over there.